src/docfx.website.themes/default/styles/docfx.js

```diff
@@ -8,6 +8,11 @@ $(function () {
   var hide = 'hide';
   var util = new utility();
 
+  // supported headers are h1, h2, h3, and h4
+  // The topest header is ignored
+  var articleSelector = ".article article";
```

Comment:
using semantic tag article is enough?

---
In reply to: [118413863](https://github.com/dotnet/docfx/pull/1695#discussion_r118413863) [](ancestors = 118413863)